rfcs/0011-longterm-storage.md

```diff
@@ -0,0 +1,105 @@
+# Long-term storage
+
+## Background
+
+Accounts act as a form of persistent storage, allowing state to persist across
+multiple transactions. To store data in accounts, however, is relatively
+expensive compared to storing transactions on the ledger. Account storage
+requires memory on *every* validator whereas transaction storage requires only
+disk space on a handful of replicators. If a client does not plan to use
```

Comment:
Replication taget could be large, like 1000.

Reply:
Sure. I suppose I can simply make the argument that disk is cheaper than memory.

Reply:
Yea, just a nit.  Handful implies a small amount of data.